client/packages/requisitions/src/RequestRequisition/DetailView/AppBarButtons/AddFromMasterListButton.tsx

```diff
@@ -1,38 +1,29 @@
 import React from 'react';
-import {
-  ButtonWithIcon,
-  useTranslation,
-  useToggle,
-  PlusCircleIcon,
-  useAuthContext,
-} from '@openmsupply-client/common';
+import { useAuthContext } from '@openmsupply-client/common';
 import { MasterListSearchModal } from '@openmsupply-client/system';
 import { useRequest } from '../../api';
 
-export const AddFromMasterListButtonComponent = () => {
-  const t = useTranslation();
-  const isDisabled = useRequest.utils.isDisabled();
-  const isProgram = useRequest.utils.isProgram();
+export const AddFromMasterListButtonComponent = ({
```

Comment:
Thoughts on `AddFromMasterListModal`? Seems to be just a modal now. Thinking we can even just call the modal directly but thats more changes haha